package.json

```diff
@@ -44,7 +44,7 @@
     "test": "jest --watch",
     "ci": "npm run lint && jest",
     "doc": "jsdoc -c .jsdoc src/*.js",
-    "doc:view": "serve -s dist/doc",
```

Comment:
This is the only actual change in the PR. I'm not sure why my other previous commits are showing up again?

Reply:
Fixed it!